apps/teams-test-app/src/components/ScrollToTop.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+
+const ScrollToTopButton = () => {
+  const [isVisible, setIsVisible] = useState(false);
+
+  // Function to check scroll position and set visibility
+  const toggleVisibility = () => {
+    if (window.scrollY > 500) {
```

Comment:
Whats your thoughts on if we instead used a variable constant for the 500? Something like `const MIN_SCROLL_POSITION`.